### PR TITLE
refactor: migrate remaining VTabs

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/pages/staking/index.vue [hunk 1]
```diff
@@ -122,28 +121,21 @@ onBeforeMount(async () => {
...
-                contain
-                src="/assets/images/protocols/kraken.svg"
-              />
-            </InternalLink>
+            <RuiTooltip v-for="item in staking" :key="item.id" open-delay="400">
+              <template #activator>
+                <InternalLink
+                  :to="Routes.STAKING.replace(':location*', item.id)"
+                >
+                  <VImg
```
Comment (kelsos) [inline frontend/app/src/pages/staking/index.vue:129]: I think this need to go after @tewshi's PR and we need to migrate this too

**Comment by kelsos** [comment]
This needs to be rebased and queued after #7085

File: frontend/app/src/pages/defi/deposits/liquidity/index.vue [hunk 2]
```diff
@@ -1,63 +1,172 @@
...
+              <template #activator>
+                <InternalLink
+                  :to="
+                    Routes.DEFI_DEPOSITS_LIQUIDITY.replace(
+                      ':location*',
+                      item.id
+                    )
+                  "
+                >
+                  <VImg
```
Comment (kelsos) [inline frontend/app/src/pages/defi/deposits/liquidity/index.vue:151]: This also has to be migrated after #7085

**Comment by kelsos** [comment]
Also small comment can we make the tab somehow in defi liquidity be highlighted for all the children?

**Comment by lukicenturi** [comment]
> Also small comment can we make the tab somehow in defi liquidity be highlighted for all the children?

Yes, something wrong with the route checking for active tab. I'll need to update the tabs component

**Comment by lukicenturi** [comment]
> Also small comment can we make the tab somehow in defi liquidity be highlighted for all the children?

updated

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**